charts/function-mesh-operator/Chart.yaml

```diff
@@ -34,3 +34,6 @@ dependencies:
     condition: admissionWebhook.enabled
     version: 0.2.6
 #    repository: https://charts.functionmesh.io/
+  - name: vertical-pod-autoscaler
```

Comment:
I'm not sure if we should add this dependency, maybe it should be added with a condition at least.

Reply:
great, that would be better

Reply:
removed the dependency